chemprop/uncertainty/evaluator.py

```diff
@@ -41,14 +44,68 @@ def evaluate(self, preds: Tensor, uncs: Tensor, targets: Tensor, mask: Tensor) -
 
 @UncertaintyEvaluatorRegistry.register("miscalibration_area")
 class CalibrationAreaEvaluator(RegressionEvaluator):
+    """
+    A class for evaluating regression uncertainty values based on how they deviate from perfect
+    calibration on an observed-probability versus expected-probability plot.
+    """
+
     def evaluate(self, preds: Tensor, uncs: Tensor, targets: Tensor, mask: Tensor) -> Tensor:
-        return
+        bin_scaling = torch.tensor([erfinv(i / 100) * np.sqrt(2) for i in range(1, 100)])
+        bin_scaling = bin_scaling.view(-1, 1, 1)
```

Comment:
also where does `100` come from?

Reply:
It is 100 because the calibration curve is plotted using 101 bin edges (i.e., 0th, 1st, ..., 100th percentiles). It is hard-coded in v1, but I am considering whether we should provide it as an attribute when initializing create the evaluator.

Reply:
I think it wouldn't be a bad idea to allow this to be parameterized via a `num_bins: int = 100` parameter. At the very least, assign `NUM_BINS = 100` as a global variable. More generally, try to avoid magic numbers when the meaning isn't clear.